server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -77,6 +79,20 @@ def split(
         )
         return StepContext(query, columns + splitted_cols)
 
+    def formula(
+        self: Self,
+        *,
+        builder: 'QueryBuilder',
+        prev_step_name: str,
+        columns: list[str],
+        step: "FormulaStep",
+    ) -> StepContext:
+        query: "QueryBuilder" = self.QUERY_CLS.from_(prev_step_name).select(
+            *columns, LiteralValue(handle_zero_division(step.formula)).as_(step.new_column)
+        )
+
+        return StepContext(query, columns + [step.new_column])
+
```

Comment:
Is this still required ?

Reply:
https://github.com/ToucanToco/weaverbird/pull/1425/commits/36e3f7b11634bdfdfdb8540552291e9762c697ad